### PR TITLE
[TestingGallery] added feature to upload last modified file from /sdcard/Wire

### PR DESCRIPTION
**Pull request by red-avtovo**
#### APK
[Download build #8669](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8669/artifact/build/artifact/wire-dev-PR725-8669.apk)
[Download build #8680](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8680/artifact/build/artifact/wire-dev-PR725-8680.apk)
[Download build #8681](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8681/artifact/build/artifact/wire-dev-PR725-8681.apk)
[Download build #8682](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8682/artifact/build/artifact/wire-dev-PR725-8682.apk)

### CODE REVIEW

File: testing_gallery/src/main/java/com/wire/testinggallery/DocumentResolver.java [hunk 1]
```diff
@@ -20,40 +20,45 @@
...
 
+import java.io.File;
+
 public class DocumentResolver {
 
-    private static final String[] ID_COLUMNS = new String[] {
-        MediaStore.Files.FileColumns._ID
-    };
+    private static final File WIRE_TESTING_FILES_DIRECTORY =
+        Environment.getExternalStoragePublicDirectory("Wire");
```
Comment (mykola-mokhnach) [inline testing_gallery/src/main/java/com/wire/testinggallery/DocumentResolver.java]: can you please rename it to "wire" and move to a constant?

File: testing_gallery/src/main/java/com/wire/testinggallery/DocumentResolver.java [hunk 2]
```diff
@@ -67,4 +72,18 @@ private Uri query(Uri baseUri, String[] projection) {
 
         return null;
     }
+
+    private Uri fileQuery(File baseDir) {
+        File choice = null;
+        long lastMod = 0;
+        for (File file : baseDir.listFiles()) {
```
Comment (mykola-mokhnach) [inline testing_gallery/src/main/java/com/wire/testinggallery/DocumentResolver.java]: just skip this stuff with last modified to save time. Simply get the first file from this folder

Comment (red-avtovo) [inline testing_gallery/src/main/java/com/wire/testinggallery/DocumentResolver.java]: ok

File: testing_gallery/src/main/java/com/wire/testinggallery/DocumentResolver.java (hunk 1 above)
Comment (red-avtovo) [inline testing_gallery/src/main/java/com/wire/testinggallery/DocumentResolver.java]: ok

**Comment by red-avtovo** [comment]
build again

**Comment by red-avtovo** [comment]
build again

**Pull request merged**